src/components/ApplicationGroup/Details/TriggerView/BulkCDTrigger.tsx

```diff
@@ -173,15 +201,122 @@ export default function BulkCDTrigger({
         if (isLoading) {
             return <Progressing pageLoader />
         }
+
+        const updateCurrentAppMaterial = (matId:number, releaseTags?:ReleaseTag[], imageComment?:ImageComment) => {
+            let updatedCurrentApp = selectedApp
+            updatedCurrentApp?.material.forEach((mat)=>{
+                if(mat.id === matId){
+                    if(releaseTags)mat.imageReleaseTags = releaseTags
+                    if(imageComment)mat.imageComment = imageComment
+                }
+            })
+            updatedCurrentApp && setSelectedApp(updatedCurrentApp)
+        }
+
         const _currentApp = appList.find((app) => app.appId === selectedApp.appId) ?? ({} as BulkCDDetailType)
+        uniqueReleaseTags.sort((a, b) => a.localeCompare(b))
+        let tagsList = ['latest']
+        tagsList.push(...uniqueReleaseTags)
```

Comment:
instead of spread and map we can directly use for of and get the same result in 1 iteration